### ISSUE TITLE
Suggestion - Default icon name

### ISSUE THREAD

**Issue by AndreThlckr** [chasemsutton/StreamDeckIconManager#2]
Hi. This app is great, but I have a suggestion that I believe would improve usability a lot.

I downloaded some icon packs online, and they have hundreds of icons, so naming each one of them one by one is not easy.

When adding multiple icons at once and skipping naming, the default name for the icon is "Custom Icon X", so it will become hard to search on the Stream Deck. If the default name could be the file's own name, it would be a lot more useful.

**Comment by alexschwantes** [comment]
Yes I second this. also note that some icons are named `custom-icon-x` so splitting the names on `-` (or other common characters) would be good.

**Comment by ChristianMalazarte** [comment]
Okay so this post is from 2022, that means there's no chance this is gonna be implemented.

**Comment by eNTy** [comment]
This has now been implemented by me, see pull request: https://github.com/chasemsutton/StreamDeckIconManager/pull/4
If not merged, just get it from me directly: https://github.com/eNTy/StreamDeckIconManager

**Comment by szynski** [comment]
> This has now been implemented by me, see pull request: [#4](https://github.com/chasemsutton/StreamDeckIconManager/pull/4) If not merged, just get it from me directly: https://github.com/eNTy/StreamDeckIconManager

It looks merged right now yet. Could you please write how to use it? Icons not named during importing still has name "Custom Icon [nr]"
I already checked your fork and oryginal source.
Thanks!

**Comment by chasemsutton** [comment]
> > This has now been implemented by me, see pull request: [#4](https://github.com/chasemsutton/StreamDeckIconManager/pull/4) If not merged, just get it from me directly: https://github.com/eNTy/StreamDeckIconManager
> 
> It looks merged right now yet. Could you please write how to use it? Icons not named during importing still has name "Custom Icon [nr]" I already checked your fork and oryginal source. Thanks!

I believe version 1.1.0.1 fixes this issue. If it is not resolved, open another issue. I recommend you uninstall any old versions before installing to avoid issues.
